src/main/java/useCases/ScheduleEvent/ScheduleEventInputData.java

```diff
@@ -0,0 +1,40 @@
+package useCases.ScheduleEvent;
+
+import entities.Tag;
+import entities.Task;
+import entities.User;
+
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.List;
+
+public class ScheduleEventInputData {
+
+    private LocalDateTime start_time;
+    private LocalDateTime end_time;
+    private List<String> collaborator_usernames;
+    private Task task;
+    private String name;
+    private boolean repeats;
+    private LocalTime frequency;
+    private List<Tag> tags;
+
+    public ScheduleEventInputData(LocalDateTime start_time,
+                                  LocalDateTime end_time,
+                                  List<String> collaborator_usernames,
+                                  Task task,
```

Comment:
give task a default value as it's optional?

Reply:
will be null if it isn't specified